tests/services/apiService.test.js

```diff
@@ -40,3 +40,54 @@ describe("fetchQuestCategoryData function", () => {
     expect(result).toEqual(mockResponse);
   });
 });
+
+describe("getQuestBoostClaimParams function", () =>  {
+
+  beforeEach(() => {
+      fetch.mockClear();
+  }  
+  );
+  
+  it("should fetch and return data for a valid  id and address", async () => {
+         const mockData = ["string", "string"]; 
+         fetch.mockResolvedValueOnce({
+          json: () => Promise.resolve(mockData),
+        });
+  
+         const result = await getQuestBoostClaimParams ("string", "address");
```

Comment:
i dont think this is a valid address and id. Please check the api response and then decide the mockdata and errors